metadata.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: clusterctl.cluster.x-k8s.io/v1alpha4
+apiVersion: clusterctl.cluster.x-k8s.io/v1alpha3
```

Comment:
why alpha3?

Reply:
They didn't change the version of this apiGroup, not sure why.

I had it confirmed by get this fixed in CAPZ: https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/1235